### PR TITLE
Dockerize the sample

### PR DESCRIPTION
**Pull request by gayashanbc**
$subject

### CODE REVIEW

File: run-docker-sample.sh [hunk 1]
```diff
@@ -18,26 +18,51 @@
 #
 # ------------------------------------------------------------------------
 
+service_provider_name=tomcat-saml-agent-sample
```
Comment (darshanasbg) [inline run-docker-sample.sh:21]: Add a description on what this script does

File: run-docker-sample.sh [hunk 2]
```diff
@@ -18,26 +18,51 @@
...
+--data-raw '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsd="http://org.apache.axis2/xsd" xmlns:xsd1="http://model.common.application.identity.carbon.wso2.org/xsd" xmlns:xsd2="http://script.model.common.application.identity.carbon.wso2.org/xsd">    <soapenv:Header/>    <soapenv:Body>       <xsd:createApplication>          <xsd:serviceProvider>             <xsd1:applicationName>'"$service_provider_name"'</xsd1:applicationName>             <xsd1:inboundAuthenticationConfig>             <xsd1:inboundAuthenticationRequestConfigs>               <xsd1:inboundAuthKey>sample-app</xsd1:inboundAuthKey>               <xsd1:inboundAuthType>samlsso</xsd1:inboundAuthType>             </xsd1:inboundAuthenticationRequestConfigs>             </xsd1:inboundAuthenticationConfig>          </xsd:serviceProvider>       </xsd:createApplication>    </soapenv:Body> </soapenv:Envelope>' \
```
Comment (darshanasbg) [inline run-docker-sample.sh:40]: Extract the soap message to a separate variable.

File: run-docker-sample.sh [hunk 3]
```diff
@@ -18,26 +18,51 @@
...
+> /dev/null
 
-curl -kv --location --request POST 'https://localhost:9443/services/IdentitySAMLSSOConfigService' \
+echo "4. Retrieveing Service Provider ID"
+service_provider_id=$(curl -sk --location --request POST 'https://localhost:9443/services/IdentityApplicationManagementService' \
+--header 'SOAPAction: getApplication' \
+--header 'Content-Type: text/xml' \
+--header 'Authorization: Basic YWRtaW46YWRtaW4=' \
```
Comment (darshanasbg) [inline run-docker-sample.sh:47]: use -u flag and lets refer default username password as primary variables defined in the beginning

File: run-docker-sample.sh (hunk 1 above)
Comment (gayashanbc) [reply run-docker-sample.sh:21]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/13/commits/d14f65689a5380fe93f51e02ff4e1c77e255c505

File: run-docker-sample.sh (hunk 2 above)
Comment (gayashanbc) [reply run-docker-sample.sh:40]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/13/commits/d14f65689a5380fe93f51e02ff4e1c77e255c505

File: run-docker-sample.sh (hunk 3 above)
Comment (gayashanbc) [reply run-docker-sample.sh:47]: Resolved in https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/13/commits/d14f65689a5380fe93f51e02ff4e1c77e255c505

**Comment by gayashanbc** [comment]
Closing this PR as the content was separated into #12 and #13

**Pull request closed without merging**